xml/geo_manage_i.xml

```diff
@@ -164,25 +197,115 @@ booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
      stopped or demoted cleanly.
     </para>
    </step>
-   <step>
+  <!-- <step>
     <para>
      Revoke <literal>&ticket1;</literal> from site
      <literal>&cluster1;</literal> with:
     </para>
 <screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 &ticket1;</screen>
-   </step>
+   </step>-->
```

Comment:
We need to explicitly revoke the ticket  first before granting it again. @ChrisKowalczyk confirmed so :)

Reply:
OK, changed.